docs/docs/pyproject.md

```diff
@@ -173,6 +173,26 @@ include = ["CHANGELOG.md"]
 exclude = ["my_package/excluded.py"]
 ```
 
+## dev-only
+
+If set to `true`, this option entirely disables installing runtime dependencies in the development environment.
```

Comment:
"If set to `true`, no production dependencies will be installed when running `poetry install`."